app/core/src/main/java/com/fsck/k9/widget/list/MessageListWidgetProvider.java

```diff
@@ -38,10 +38,19 @@ public void onUpdate(Context context, AppWidgetManager appWidgetManager, int[] a
         }
     }
 
+    //----------------------------------change LB 29.06.2018-------------------------------------
```

Comment:
We don't want comments like this in our code base. In the future please remove them before submitting a pull request.